### ISSUE TITLE
Cannot access the channel log from Pidgin's UI. 

### ISSUE THREAD

**Issue by fangfufu** [EionRobb/purple-discord#296]
I unticked "Auto-create rooms on buddy list", manually added the Discord text channels. I am receiving messages fine. However if I right click on the chat room, then click "View Log", it says "No logs were found". I went into my Pidgin's log folder, it is logging the channel fine. So clearly something went wrong at the UI.

**Comment by EionRobb** [comment]
Can you attach a screenshot?  Is it for all historical messages from before you were using Pidgin that you're expecting to see, or are the logs from the past 24 hrs not showing up either?

eg, this is what it looks for me in the Unofficial Pidgin channel support channel:
![image](https://user-images.githubusercontent.com/1063865/85235934-4af18a80-b46d-11ea-8897-a840e50a7af8.png)

**Comment by fangfufu** [comment]
I just get "No logs were found", when I try opening up the log file. The window in the left hand side is the chat window. 
![image](https://user-images.githubusercontent.com/2323403/85236713-3040f680-b418-11ea-85cb-e98f266a4ff4.png)

But it logged everything properly if I go the relevant log folder ~/.purple/logs/discord/something@something.com/a_random_number.chat

Here is a screenshot of the log. 
![image](https://user-images.githubusercontent.com/2323403/85236799-fae8d880-b418-11ea-8f05-5040d017bd16.png)

**Comment by EionRobb** [comment]
I wonder if it's to do with the manually created channels.  When you added them, did you Conversation->Add Chat, or did you use the Tools->Room List?

When you right-click on the chat in the buddy list and choose "Edit Settings", what shows up?

eg, for the Pidgin support channel its:
![image](https://user-images.githubusercontent.com/1063865/85237175-d7557a80-b478-11ea-9824-81571632352c.png)

**Comment by fangfufu** [comment]
Yes, you are right, they were all manually added, because I only want to monitor a few channels. :) 
![image](https://user-images.githubusercontent.com/2323403/85237596-fa077500-b41f-11ea-8caa-4766b4ec9f29.png)

**Comment by EionRobb** [comment]
And does the ID in the 'Edit Chat' popup match the `a_random_number` in 
 `~/.purple/logs/discord/something@something.com/a_random_number.chat`?

**Comment by fangfufu** [comment]
Yup, it certainly does. The logs are definitely being generated properly as well.

**Comment by Hawk777** [comment]
I noticed that if you copy the ID into the Name field, then View Log starts working properly. It doesn’t seem to affect the name displayed to the user; that comes from the Alias instead. So I’ve been going through my channels and setting the Name equal to the ID so I can access logs. I have no idea what the Name is even used for; AFAICT it doesn’t show up anywhere at all.

**Comment by chainria** [comment]
This is a very old bug which has a quite simple solution: Access the logs from within the channel itself. Then it works. I don't really know why exactly that is, but maybe this is of help to someone!

**Comment by alexolog** [comment]
> I noticed that if you copy the ID into the Name field, then View Log starts working properly. It doesn’t seem to affect the name displayed to the user; that comes from the Alias instead.

As @Hawk777 said, use the same numeric value for both fields and change the display name with an alias, otherwise some things will break.

**Comment by Hawk777** [comment]
> > I noticed that if you copy the ID into the Name field, then View Log starts working properly. It doesn’t seem to affect the name displayed to the user; that comes from the Alias instead.
> 
> As @Hawk777 said, use the same numeric value for both fields and change the display name with an alias, otherwise some things will break.

To clarify, I had to *change* my channels to meet this configuration. They didn’t come that way by default (by default the Name field was set to the channel name, not the ID number). Once I changed them, then logs started working.

**Comment by chainria** [comment]
> To clarify, I had to _change_ my channels to meet this configuration. They didn’t come that way by default (by default the Name field was set to the channel name, not the ID number). Once I changed them, then logs started working.

You're right! I tried changing a random channel and pasting the ID in both "ID" and "Name" fields. And now I can show logs. @EionRobb I think there is something wrong with BL population then.
